### PR TITLE
Improved dev config

### PR DESCRIPTION
**Pull request by ariansobczak-rst**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -17,9 +17,7 @@
   "[jsonc]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
-  "editor.codeActionsOnSave": {
-    "source.fixAll": true
-  },
+  "editor.codeActionsOnSave": { "source.fixAll": true },
```
Comment (dziraf) [inline .vscode/settings.json:20]: I wonder if auto save fires during i. e. typescript build when new js files are created? Unless dist folder is ignored, I'm not sure

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "incremental": true,
+    "lib": ["esnext"],
+    "module": "CommonJS",
+    "moduleResolution": "Node",
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "outDir": "./dist",
+    "paths": { "@monorepo/*": ["packages/*/src/index.ts"] },
```
Comment (dziraf) [inline tsconfig.json:15]: `@monorepo` ?

File: .vscode/settings.json (hunk 1 above)
Comment (ariansobczak-rst) [reply .vscode/settings.json:20]: This setting works only in the editor so wouldn't be execute during ts build

Comment (dziraf) [reply .vscode/settings.json:20]: Nvm, it's vscode specific

**Review by dziraf — APPROVED** [review]

**Pull request merged**